crates/store/re_types/definitions/rerun/blueprint/archetypes/map_options.fbs

```diff
@@ -0,0 +1,18 @@
+namespace rerun.blueprint.archetypes;
+
+
+/// Configuration for the background of a view.
+table MapOptions (
```

Comment:
I'm not entirely sure if the grouping of "map options" makes sense like this, but I don't have a better idea right now either.
Usually we try to group such that the generic blueprint api & ui becomes meaningful. This now just generates "options" which isn't amazing

Reply:
Right. In the future, we definitely want to expose the map center location or something similar, so that will be grouped with `ZoomLevel`. We can definitely split of the map provider though. Maybe there will be other options in the future falling in the "background" category.

Reply:
Let's go for:
- `area` / `MapArea` / `ZoomLevel` (in the future something like `CenterPosition` TBD)
- `background` / `MapBackground` / `Provider`

Reply:
Provider seems more fundamental than the others since instead of styling it's more "data content", but otherwise make sense!